packages/gen-ai-hub/src/orchestration/orchestration-types.ts

```diff
@@ -39,16 +41,48 @@ export interface PromptConfig {
  */
 export type LlmConfig = LLMModuleConfig;
 
+/**
+ * Wrapper object to configure Filters.
+ */
+export interface FilterConfig {
+  /**
+   * Input configuration for filtering provider.
+   */
+  input?: FilterServiceProvider;
```

Comment:
We could consider:
```suggestion
  input?: FilterServiceProvider | [FilterServiceProvider];
```

That would also allow:
```
filterConfig: {
        input: [] as FilterServiceProvider,
```

Not sure how to best implement this internally, but using `if (Array.isArray(config.input))` was what copilot suggested me 😉 

Only downside I can think of is that the type signature becomes a bit more complex.